s3/pkg/meta/db/drivers/tidb/object.go

```diff
@@ -34,12 +34,12 @@ func (t *TidbClient) GetObject(ctx context.Context, bucketName, objectName, vers
 	var row *sql.Row
 	if version == "" {
 		sqltext = "select bucketname,name,version,location,tenantid,userid,size,objectid,lastmodifiedtime,etag," +
-			"contenttype,customattributes,acl,nullversion,deletemarker,ssetype,encryptionkey,initializationvector,type,tier,storageMeta" +
+			"contenttype,customattributes,acl,nullversion,deletemarker,ssetype,encryptionkey,initializationvector,type,tier,storageMeta,encsize" +
```

Comment:
It seems the schema of objects table in the database is changed, so the database initiation, in tidb.sql, also need to be changed accordingly.

Reply:
fixed